.github/release-please.yml

```
@@ -1,3 +1,4 @@
 handleGHRelease: true
 releaseType: go
 releaseLabel: "autorelease: published"
+bump-minor-pre-major: true
```

```suggestion
bumpMinorPreMajor: true
```
Didn't know release-please had a schema linter. Nice! Let's see if this is valid.